### PR TITLE
Category tree broken

### PR DESCRIPTION
**Pull request by btry**
#2467 is incomplete, browsing categories is broken

### CODE REVIEW

**Comment by Rom1-B** [comment]
Text is cut off in the list:

image

**Review by Rom1-B — COMMENTED** [review]
When setting the "merged with forms" entity, there is an error with "see all":
image
But it works with the "distinct menu entry" parameter.

File: ajax/homepage_wizard.php [hunk 1]
```diff
@@ -53,7 +53,7 @@
 }
 
 function plugin_formcreator_showWizardCategories() {
-   $tree = PluginFormcreatorCategory::getCategoryTree(0, false);
+   $tree = PluginFormcreatorCommon::getCategoryTree(0, false);
```
Comment (Rom1-B) [inline ajax/homepage_wizard.php]:
```suggestion
   $tree = PluginFormcreatorCommon::getCategoryTree(0, false);
```
Maybe change also in `ajax/kb_category.php`

**Comment by AdrienClairembault** [comment]
If I understand correctly, form categories have been removed and we now use only kb categories.
It seems unusual to have "Forms" in a "Knowledge base category".

Shouldn't the item be renamed to something more generic ("Category"  ?).

**Review by AdrienClairembault — APPROVED** [review]
Functionally seems OK, category tree is working correctly.

**Comment by btry** [comment]
The service catalog was designed to merge FAQ and forms, helping the user to find a solution to his problem from FAQ before filling an assistance request. Then dropping form categories and the not very convenient relation between form categories and KB categories seems to me a good thing. 

I think "category" would be to much generic (one could ask "category of what ?"), and this change would be in GLPI core, might break some other plugin.

**Comment by AdrienClairembault** [comment]
> The service catalog was designed to merge FAQ and forms, helping the user to find a solution to his problem from FAQ before filling an assistance request. Then dropping form categories and the not very convenient relation between form categories and KB categories seems to me a good thing.

Yes this is a good change 👍 .

> I think "category" would be to much generic (one could ask "category of what ?"), and this change would be in GLPI core, might break some other plugin.

I was thinking of changing the displayed name only, not the class name.
Shouldn't break anything.
I agree "Category" is a very generic... how about something like "Helpdesk category" ?

It just seems badly named right now as it does not contain only Knowledge base items.

**Pull request merged**